app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -323,19 +323,19 @@ ListingController = (
 
   $scope.priorityLabel = (priority, modifier) ->
     labelMap =
-      'Vision impaired':
+      'Vision impairments':
         name: 'Vision Impairments'
         description: 'impaired vision'
-      'Hearing impaired':
+      'Hearing impairments':
         name: 'Hearing Impairments'
         description: 'impaired hearing'
-      'Hearing/Vision impaired':
+      'Hearing/Vision impairments':
         name: 'Vision and/or Hearing Impairments'
         description: 'impaired vision and/or hearing'
-      'Mobility/hearing/vision impaired':
+      'Mobility/hearing/vision impairments':
         name: 'Mobility, Hearing and/or Vision Impairments'
```

Comment:
@callaghanc after looking at this function more closely, there is actually a change in the name as well. It takes what is returned from salesforce and makes it a bit clearer with the "and/or"s